opensuse_buildservice/debian/postinst

```diff
@@ -0,0 +1,102 @@
+#!/bin/sh
+# postinst script for notqmail
+#
+# see: dh_installdeb(1)
+
+set -e
+
+# summary of how this script can be called:
+#        * <postinst> `configure' <most-recently-configured-version>
+#        * <old-postinst> `abort-upgrade' <new version>
+#        * <conflictor's-postinst> `abort-remove' `in-favour' <package>
+#          <new-version>
+#        * <postinst> `abort-remove'
+#        * <deconfigured's-postinst> `abort-deconfigure' `in-favour'
+#          <failed-install-package> <version> `removing'
+#          <conflicting-package> <version>
+# for details, see http://www.debian.org/doc/debian-policy/ or
+# the debian-policy package
+
+qmaildir=/var/qmail
+logdir=/var/log/svc
+cp=/bin/cp
+rm=/bin/rm
+mv=/bin/mv
+####################################
+
+ID=`id -u`
+if [ $ID -ne 0 ] ; then
+	echo "You are not root" >&2
+	exit 1
+fi
+
+case "$1" in
+    configure)
+		if [ " $2" = " " ] ; then
+		echo "installing notqmail"
+		else
+		echo "upgrading notqmail $2"
+		fi
+    ;;
+
+    abort-upgrade|abort-remove|abort-deconfigure)
+	exit 0
+    ;;
+
+    *)
+        echo "postinst called with unknown argument \`$1'" >&2
+        exit 1
+    ;;
+esac
+
+# dh_installdeb will replace this with shell code automatically
+# generated by other debhelper scripts.
+
+#DEBHELPER#
+
+# notqmail.upgrade/notqmail.install are created
+# by the preinst script
+if [ -f /tmp/notqmail.upgrade ] ; then # upgrade
+  if [ -x /bin/systemctl -a -d /lib/systemd/system ] ; then
+    cmp -s ${qmaildir}/boot/notqmail.service /lib/systemd/system/notqmail.service >/dev/null 2>&1 || true
+    ${cp} ${qmaildir}/boot/notqmail.service /lib/systemd/system/notqmail.service
+    /bin/systemctl daemon-reload
+  elif [ -x /bin/systemctl -a -d /usr/lib/systemd/system ] ; then
+    cmp -s ${qmaildir}/boot/notqmail.service /usr/lib/systemd/system/notqmail.service >/dev/null 2>&1 || true
+    ${cp} ${qmaildir}/boot/notqmail.service /usr/lib/systemd/system/notqmail.service
+    /bin/systemctl daemon-reload
+  fi
+  ${rm} -f /tmp/notqmail.upgrade
+  exit 0
+fi
+${rm} -f /tmp/notqmail.install
+
+echo "Post Installation..."
+(
+echo "Creating ${logdir}"
+if [ ! -d ${logdir} ] ; then
```

Comment:
Is this check needed? ```mkdir -p``` will just do nothing if the directory already exists.

Reply:
Yes